src/OOVPADatabase/D3D8LTCG/3911.inl

```diff
@@ -905,18 +905,11 @@ OOVPA_END;
 // * D3DDevice_SetTexture
 // ******************************************************************
 //81C10000F8FF89 ...C20400
-OOVPA_NO_XREF(D3DDevice_SetTexture_4, 2024, 9)
+OOVPA_NO_XREF(D3DDevice_SetTexture_4__LTCG_eax_pTexture, 2024, 9)
 
-        { 0x00, 0x83 },
-        { 0x01, 0xEC },
-
-        { 0x27, 0x81 },
-        { 0x28, 0xC1 },
-        { 0x29, 0x00 },
-        { 0x2A, 0x00 },
-        { 0x2B, 0xF8 },
-        { 0x2C, 0xFF },
-        { 0x2D, 0x89 },
+        OV_MATCH(0x00, 0x83, 0xEC), // sub esp, ?
+        OV_MATCH(0x1A, 0x8B, 0xF0), // mov esi, eax
+        OV_MATCH(0x27, 0x81, 0xC1, 0x00, 0x00, 0xF8, 0xFF, 0x89), // add ecx, 0FFF80000h / mov ?
```

Comment:
```suggestion
        // sub esp, ...
        OV_MATCH(0x00, 0x83, 0xEC),

        // mov esi, eax
        OV_MATCH(0x1A, 0x8B, 0xF0),

        // add ecx, 0xFFF80000; mov ...
        OV_MATCH(0x27, 0x81, 0xC1, 0x00, 0x00, 0xF8, 0xFF, 0x89),
```
See reference from [wiki page of how to write it](https://github.com/Cxbx-Reloaded/XbSymbolDatabase/wiki/Maintaining-OOVPAs-for-Symbol-detection#too-long-didnt-read). Wiki's example of symbol + offset in the comment section is not a requirement.